app/services/git/fetches_updated_repos.rb

```diff
@@ -18,31 +18,29 @@ def run
   end
 
   def fetch
-    find_repo_update
-    fetch_repo_update if repo_update
+    repo_updates.each { |update| FetchRepoUpdateJob.perform_now(update.id) }
```

Comment:
Is there a potential need for de-duping fetches at some point? Say if there are N hook events for the same repo in rapid succession?